Binary files rpmbuild-orig/SOURCES/gcc48-4.8.2-15.el6.0.src.rpm and rpmbuild/SOURCES/gcc48-4.8.2-15.el6.0.src.rpm differ
diff -ruN rpmbuild-orig/SPECS/firefox.spec rpmbuild/SPECS/firefox.spec
--- rpmbuild-orig/SPECS/firefox.spec	2016-01-28 15:08:54.000000000 +0100
+++ rpmbuild/SPECS/firefox.spec	2016-02-13 14:39:38.649388254 +0100
@@ -52,7 +52,7 @@
 Summary:        Mozilla Firefox Web browser
 Name:           firefox
 Version:        38.6.0
-Release:        2%{?prever}%{?dist}
+Release:        2%{?prever}%{?dist}.0
 URL:            http://www.mozilla.org/projects/firefox/
 License:        MPLv1.1 or GPLv2+ or LGPLv2+
 Group:          Applications/Internet
@@ -71,7 +71,7 @@
 Source24:       mozilla-api-key
 Source100:      find-external-requires
 Source200:      https://www.python.org/ftp/python/2.7.8/Python-2.7.8.tgz
-Source300:      gcc48-%{gcc_version}.el6.src.rpm
+Source300:      gcc48-%{gcc_version}.el6.0.src.rpm
 
 # Build patches
 Patch0:         firefox-install-dir.patch
@@ -682,6 +682,9 @@
 #---------------------------------------------------------------------
 
 %changelog
+* Sat Feb 13 2016 Jacco Ligthart <jacco@redsleeve.org> - 38.6.0-2.0
+- fixed the bundled gcc to compile on arm (thanks Gordan)
+
 * Wed Jan 27 2016 Jan Horak <jhorak@redhat.com> - 38.6.0-2
 - Added 'loop.enabled' to default config
 
